addon/formatter-base.js

```diff
@@ -45,7 +27,8 @@ var FormatBase = Ember.Object.extend({
 });
 
 FormatBase.reopenClass({
-    formatOptions: Ember.A()
+    formatOptions: Ember.A(['locales', 'format']),
```

Comment:
treats locales and format as allowable arguments that get passed on through to the formatter constructors as the second argument.